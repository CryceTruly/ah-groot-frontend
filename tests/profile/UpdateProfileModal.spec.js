import React from 'react';
import { shallow } from 'enzyme';

import UpdateProfileModal from '../../src/components/profile/profileModal';

describe('UpdateProfileModal', () => {
 
  it('should render without crashing', () => {
    const wrapper = shallow(<UpdateProfileModal />);
    expect(wrapper).toMatchSnapshot();
  });
});