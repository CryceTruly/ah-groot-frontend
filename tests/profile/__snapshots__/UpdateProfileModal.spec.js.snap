// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UpdateProfileModal should render without crashing 1`] = `ShallowWrapper {}`;
